src/cli/update.rs

```diff
@@ -0,0 +1,386 @@
+                    match package {
+                        Package::Conda(p) => {
+                            diff.removed.push(Package::Conda(p));
+                        }
+                        Package::Pypi(p) => {
+                            diff.removed.push(Package::Pypi(p));
+                        }
+                    }
+                }
+                environment_diff.insert(platform, diff);
+            }
+
+            // Remove empty diffs
+            environment_diff.retain(|_, diff| !diff.is_empty());
+
+            result
+                .environment
+                .insert(environment_name.to_string(), environment_diff);
+        }
+
+        // Find environments that were completely removed
+        for (environment_name, environment) in previous
+            .environments()
+            .filter(|(name, _)| !result.environment.contains_key(*name))
+            .collect_vec()
+        {
+            let mut environment_diff = IndexMap::new();
+            for (platform, packages) in environment.packages_by_platform() {
+                let mut diff = PackagesDiff::default();
+                for package in packages {
+                    match package {
+                        Package::Conda(p) => {
+                            diff.removed.push(Package::Conda(p));
+                        }
+                        Package::Pypi(p) => {
+                            diff.removed.push(Package::Pypi(p));
+                        }
+                    }
+                }
+                environment_diff.insert(platform, diff);
+            }
+            result
+                .environment
+                .insert(environment_name.to_string(), environment_diff);
+        }
+
+        // Remove empty environments
+        result.environment.retain(|_, diff| !diff.is_empty());
+
+        result
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.environment.is_empty()
+    }
+
+    // Format the lock-file
+    pub fn print(&self) {
+        enum Change<'i> {
+            Added(&'i Package),
+            Removed(&'i Package),
+            Changed(&'i Package, &'i Package),
+        }
+
+        fn format_package_identifier(package: &Package) -> String {
+            match package {
+                Package::Conda(p) => format!(
+                    "{} {}",
+                    &p.package_record().version.as_str(),
+                    &p.package_record().build
+                ),
+                Package::Pypi(p) => p.data().package.version.to_string(),
+            }
+        }
+
+        for (environment_name, environment) in
+            self.environment.iter().sorted_by(|(a, _), (b, _)| a.cmp(b))
+        {
+            println!(
+                "Environment: {}",
+                consts::ENVIRONMENT_STYLE.apply_to(environment_name),
+            );
+            for (platform, packages) in environment {
+                println!("  Platform: {}", consts::PLATFORM_STYLE.apply_to(platform));
```

Comment:
```suggestion
            for (platform, packages) in environment {
                println!("Environment: {} {}", consts::ENVIRONMENT_STYLE.apply_to(environment_name), consts::PLATFORM_STYLE.apply_to(platform));
```

I was often looking at the list like this:
![image](https://github.com/prefix-dev/pixi/assets/12893423/b25b21a5-d859-47fc-b9bf-f707f314147a)

I think it wouldn't overload the information and could help with quickly seeing what you're looking at.